src/Resource.js

```diff
@@ -308,15 +308,26 @@ Resource.prototype._loadImage = function () {
  * @private
  */
 Resource.prototype._loadElement = function (type) {
-    this.data = document.createElement(type);
+    if (type === 'audio' && typeof Audio !== 'undefined') {
```

Comment:
Yes, it breaks up. Probably a basic check could be added, it might help older browsers as well:

```javascript
    if (this.data === null) {
        this.error = new Error('Unsupported element ' + type);
        this.complete();
        return;
    }
```